cpp/krim_ssoar.sh

```diff
@@ -3,7 +3,8 @@
 set -e
 
 echo "Download the data from SSOAR"
-oai_pmh_harvester https://www.ssoar.info/OAIHandler/request marcxml col_collection_10214 KRIM_SSOAR krim_ssoar.xml 20
+oai_pmh_harvester https://www.ssoar.info/OAIHandler/request marcxml col_collection_10214 KRIM_SSOAR \
```

Comment:
Do we need to migrate any existing dups db as soon as we activate this?

Reply:
Done.

Reply:
This does not seem to match the new usage --path-to-dups-database.

Reply:
Fixed.